src/SFA.DAS.RoATPService.Application.Api/Controllers/OrganisationController.cs

```diff
@@ -45,14 +45,15 @@ public async Task<IActionResult> Get(Guid organisationId)
         [SwaggerResponse((int)HttpStatusCode.OK, Type = typeof(IEnumerable<Engagement>))]
```

Comment:
Do we want to keep an API endpoint to return all engagements?

Reply:
Yes. My understanding although is not explicitly mentioned in the ticket is that we do.

Reply:
Yes. I'm fine with this.